backend/app/modules/octant_rewards/controller.py

```diff
@@ -17,3 +21,10 @@ def get_leverage(epoch_num: int) -> float:
         raise NotImplementedForGivenEpochState()
     service = get_services(context.epoch_state).octant_rewards_service
     return service.get_leverage(context)
+
+
+def get_last_epoch_leverage() -> float:
+    current_epoch_num = epochs.get_current_epoch()
```

Comment:
I wonder if we should take the leverage from epoch that is `pending` or we should take into account only `finalized`. Until it stabilises after a decent amount of allocations, the result will be huge for `pending`. But maybe it's not that bad, because it can encourage to allocate

Reply:
There is nothing to wonder. It should be taken from `finalized`, [as Matt requested in the ticket (see screenshot of my conversation with him](https://linear.app/golemfoundation/issue/OCT-1542/rewards-calculator-update-to-improve-results-server).